test/utils-specs.js

```diff
@@ -178,4 +179,65 @@ describe('utils', function () {
       });
     });
   });
+
+  describe('pullSettings()', function () {
```

Comment:
what about a test for a setting that has an object as a value?

Reply:
sure, will add